music21/musicxml/test_m21ToXml.py

```diff
@@ -413,6 +413,71 @@ def testMultiDigitEndingsWrite(self):
         endings = x.findall('.//ending')
         self.assertEqual([e.get('number') for e in endings], ['', '', '3', '3'])
 
+    def testMultiMeasureEndingsWrite(self):
+        # Relevant barlines:
+        # Measure 1, left barline: no ending
+        # Measure 1, right barline: no ending
+        # Measure 2, left barline: <ending number="1" type="start"/>
+        # Measure 2, right barline: no ending
+        # Measure 3, left barline: no ending
+        # Measure 3, right barline: no ending
+        # Measure 4, left barline: no ending
+        # Measure 4, right barline: <ending number="1" type="stop"/>
+        # Measure 5, left barline: <ending number="2" type="start"/>
+        # Measure 5, right barline: no ending
+        # Measure 6, left barline: no ending
+        # Measure 6, right barline: no ending
+        # Measure 7, left barline: no ending
+        # Measure 7, right barline: <ending number="2" type="stop"/>
+        # Measure 8, left barline: no ending
+        # Measure 8, right barline: no ending
+        s = converter.parse(testPrimitive.multiMeasureEnding)
+        x = self.getET(s)
+        endings = x.findall('.//ending')
+        self.assertEqual([e.get('number') for e in endings], ['1', '1', '2', '2'])
+
+        expectedEndings = {
+            # key = measure number, value = list(leftEndingType, rightEndingType)
+            '1': [None, None],     # measure before the endings
+            '2': ['start', None],  # first measure of ending 1
+            '3': [None, None],     # second measure of ending 1
+            '4': [None, 'stop'],   # last measure of ending 1
+            '5': ['start', None],  # first measure of ending 2
+            '6': [None, None],     # second measure of ending 2
+            '7': [None, 'stop'],   # last measure of ending 2
+            '8': [None, None]      # measure after the endings
+        }
+        measures = x.findall('.//measure')
+        self.assertEqual(len(measures), 8)
+        for measure in measures:
+            measNumber = measure.get('number')
```

Comment:
```suggestion
        for measure in measures:
            measNumber = measure.get('number')
            with self.subTest(measureNumber=measNumber):
```

If any tests fail, it will turn this:
```
======================================================================
FAIL: testMultiMeasureEndingsWrite (__main__.Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jacobwalls/music21/music21/musicxml/test_m21ToXml.py", line 477, in testMultiMeasureEndingsWrite
    self.assertFalse(gotLeftBarline)
AssertionError: True is not false

----------------------------------------------------------------------
Ran 27 tests in 1.737s
```

Into this:
```
======================================================================
FAIL: testMultiMeasureEndingsWrite (__main__.Test) (measureNumber='2')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jacobwalls/music21/music21/musicxml/test_m21ToXml.py", line 478, in testMultiMeasureEndingsWrite
    self.assertFalse(gotLeftBarline)
AssertionError: True is not false

======================================================================
FAIL: testMultiMeasureEndingsWrite (__main__.Test) (measureNumber='5')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jacobwalls/music21/music21/musicxml/test_m21ToXml.py", line 478, in testMultiMeasureEndingsWrite
    self.assertFalse(gotLeftBarline)
AssertionError: True is not false

----------------------------------------------------------------------
Ran 27 tests in 1.520s

FAILED (failures=2)
```

Reply:
Done.